scripts/ci/perf_test.sh

```diff
@@ -58,18 +58,18 @@ EOF
 
 # poll for influx service ready
 set +e
-result=$(ssh ubuntu@$ec2_ip "influx ping")
-while [ "$result" != "OK" ]; do
+result=$(ssh ubuntu@$ec2_ip "curl -s -o /dev/null http://localhost:8086/ready -w %{http_code}")
+while [ "$result" != "200" ]; do
   sleep 2
-  result=$(ssh ubuntu@$ec2_ip "influx ping")
+  result=$(ssh ubuntu@$ec2_ip "curl -s -o /dev/null http://localhost:8086/ready -w %{http_code}")
 done
 set -e
 
 # setup influxdb2
 export INFLUXDB2=true
 export TEST_ORG=example_org
 export TEST_TOKEN=token
-ssh ubuntu@$ec2_ip "influx setup -c default -b benchmark_db -n default -o $TEST_ORG -p thisisnotused -r 0 -t $TEST_TOKEN -u ubuntu --skip-verify --force"
+ssh ubuntu@$ec2_ip "curl -s -H \"Content-Type: application/json\" -XPOST -d '{\"username\": \"default\", \"password\": \"thisisnotused\", \"retentionPeriodSeconds\": 0, \"retentionPeriodHours\": 0, \"org\": \"$TEST_ORG\", \"bucket\": \"unused_bucket\", \"token\": \"$TEST_TOKEN\"}' http://localhost:8086/setup -w %{http_code}"
```

Comment:
You can leave out `retentionPeriodHours` here if you want (it's deprecated / replaced by `retentionPeriodSeconds`)

Reply:
Should we check that this returns a successful code?

Reply:
With `set -e` we "get that for free" in that the ssh wrapper will return nonzero if its command does. Which is how I found out about this breakage to begin with.

Reply:
Yes, I'll fix.

Reply:
Hmm, but even the following returns a 200?

`ssh -i /home/ubuntu/.ssh/key.pem ubuntu@<public IP> 'curl -o /dev/null -XPOST -H '\''application/json'\'' http://localhost:8086/nonsense -d '\''{"junk": "nothing"}'\'' -w %{http_code}'`

Reply:
That's an unfortunate side-effect of the embedded UI, I think. Other than a few one-off things like `/ready`, any route that doesn't start with `/api/v2` is assumed to be a UI route. Since the back-end doesn't know any details about the UI it just returns the base HTML page w/ a 200 response, assuming the embedded React router in the front-end will handle parsing the route.

Reply:
Ah, so we may actually get a 50x from `/ready` even though other endpoints are set to just return 200?

Reply:
`/ready` is currently hard-coded to return a 200 from the backend, but it could hypothetically return a 50x if we ever update the impl. The call to the setup API here could also definitely return an error code.

I just noticed, the HTTP path here needs to be `/api/v2/setup` for the request to reach the back-end

Reply:
Ok, changed

Reply:
The [API docs state that /ready could return a non-2xx error](https://docs.influxdata.com/influxdb/v2.0/api/#operation/GetReady). It sounds like that wouldn't come from the endpoint itself but from the server in general if something were really borked